examples/cg/cg.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import contextlib
+import time
+
+import numpy as np
+import six
+
+import cupy
+
+
+@contextlib.contextmanager
+def timer(message):
+    cupy.cuda.Stream.null.synchronize()
+    start = time.time()
+    yield
+    cupy.cuda.Stream.null.synchronize()
+    end = time.time()
+    print('%s:  %f sec' % (message, end - start))
+
+
+def fit(A, b, tol, max_iter):
```

Comment:
I think it is better to emphasize that this function is agnostic to the environment on which the code runs. So how about adding something like "Note that this function works even tensors A and b are NumPy or CuPy arrays" as a description of `fit` or `run`?